CIScripts/Build/BuildFunctions.ps1

```diff
@@ -401,3 +401,61 @@ function Invoke-AgentTestsBuild {
 
     $Job.PopStep()
 }
+
+function Invoke-ContainerBuild {
```

Comment:
We could move `Invoke-ContainerBuild` and `Invoke-ContainersBuild` to a new file named ` CIScripts/Build/Container.ps1`.